sdk/identity/azure-identity/src/main/java/com/azure/identity/DefaultAzureCredentialBuilder.java

```diff
@@ -34,6 +36,29 @@ public DefaultAzureCredentialBuilder authorityHost(String authorityHost) {
     }
 
 
+    /**
+     * Specifies the kee pass database path to read the cached credentials of Azure toolkit for IntelliJ plugin.
+     * The {@code databasePath} is required on windows platform. For mac and linux platform native key chain /
+     * key ring will be accessed respectively to retrieve the cached credentials.
+     *
+     * <p>This path can be located in the IntelliJ IDE.
+     * Windows: File -&gt; Settings -&gt; Appearance &amp; Behavior -&gt; System Settings -&gt; Passwords. </p>
+     *
+     * @param databasePath the path to the kee pass database.
+     * @throws IllegalArgumentException if {@code databasePath is either not specified or is empty}
+     * @return An updated instance of this builder with the kee pass database path set as specified.
+     */
+    public DefaultAzureCredentialBuilder keePassDatabasePath(String databasePath) {
```

Comment:
Should this property denote that it's the KeePass database path for IntelliJ? intelliJKeyPassDatabasePath?